test/stl_concept_covering.cpp

```diff
@@ -78,6 +78,17 @@ namespace accum
   }
 }
 
+// for std::shuffle
```

Comment:
@pdimov I'm guessing I could have done this using archetypes and not an implementation of the URBG concept, but this is a new concept for me (pun not intended).